app/signin/page.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+import React, { useState } from "react";
+import styles from "../../styles/SigninPage.module.css";
+
+import { loginSuccess } from "../../utils/commonUtil";
+import { postReqeustApi } from "../../utils/requestApi";
+import SignInfoInputComponent from "../../components/SignInfoInputComponent";
+
+// email 정규식
+const regex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
+
+const SigninPage = () => {
+  const [signinInfo, setSigninInfo] = useState({
+    email: false,
+    password: false,
+  });
+
+  const [emailErrorText, setEmailErrorText] = useState("");
+  const [pwErrorText, setPwErrorText] = useState("");
```

Comment:
pw로 쓰려면 위에서도 pw로 써주는 것과 같이 일관되게 가져가는게 중요할 것 같고,
개인적으로는 축약없이 password로 표현해주시면 좋을 것 같아요~
```suggestion
    password: false,
  });

  const [emailErrorText, setEmailErrorText] = useState("");
  const [passwordErrorText, setPasswordErrorText] = useState("");
```